Formula/futhark.rb

```diff
@@ -5,6 +5,7 @@ class Futhark < Formula
   sha256 "262dd8024c58ebb05964010227bc76907c873f261ab3348ffeab5fa5b1def022"
   license "ISC"
   head "https://github.com/diku-dk/futhark.git"
+  revision 1
```

Comment:
```suggestion
  revision 1
  head "https://github.com/diku-dk/futhark.git"
```
Also, I don't actually think this needs a revision, as the change you've made is relevant only for the build.